igor-web/src/main/groovy/com/netflix/spinnaker/igor/jenkins/service/JenkinsService.java

```diff
@@ -331,12 +335,12 @@ private Response getPropertyFile(String jobName, Integer buildNumber, String fil
         () -> {
           try {
             return jenkinsClient.getPropertyFile(encode(jobName), buildNumber, fileName);
-          } catch (RetrofitError e) {
+          } catch (SpinnakerServerException e) {
```

Comment:
Similar to elsewhere, since this re-throws network errors, let's replace this with
```suggestion
          } catch (SpinnakerHttpException e) {
```